addons/l10n_it_edi/tests/test_ir_mail_server.py

```diff
@@ -158,3 +180,140 @@ def test_ricevuta_consegna(self):
     def test_decorrenza_termini(self):
         """ Test a receipt adapted from https://www.fatturapa.gov.it/export/documenti/messaggi/v1.0/IT01234567890_11111_DT_001.xml """
         self._test_receipt('DT', 'delivered', 'delivered_expired')
+
+    def test_price_included_taxes(self):
+        """ When the tax is price included, there should be a rounding value added to the xml, if the sum(subtotals) * tax_rate is not
+            equal to taxable base * tax rate (there is a constraint in the edi where taxable base * tax rate = tax amount, but also
+            taxable base = sum(subtotals) + rounding amount)
+        """
+        price_included_invoice = self.env['account.move'].create({
```

Comment:
Please move test data definition up to the setUpClass(), just like `cls.invoice_content`
https://github.com/odoo/odoo/blob/1cf1bda1dc4a33199ae9744baa5627d5c7f59cd1/addons/l10n_it_edi/tests/test_ir_mail_server.py#L60-L70